app/src/main/java/org/simple/clinic/summary/compose/StatinNudgeView.kt

```diff
@@ -0,0 +1,149 @@
+                            modifier = Modifier
+                                .background(SimpleTheme.colors.material.error, shape = RoundedCornerShape(50))
+                                .padding(horizontal = 8.dp, vertical = 4.dp),
+                            style = SimpleTheme.typography.material.button,
+                            color = SimpleTheme.colors.onToolbarPrimary,
+                            text = stringResource(R.string.statin_alert_at_risk_patient)
+                        )
+                    }
+                }
+                Spacer(modifier = Modifier.height(4.dp))
+                RiskProgressBar()
+                Spacer(modifier = Modifier.height(16.dp))
+                Text(
+                    text = buildAnnotatedString {
+                        append(stringResource(R.string.statin_alert_refer_to_doctor_for))
+                        append(" ")
+                        withStyle(style = SimpleTheme.typography.body2Bold.toSpanStyle()) {
+                            append(stringResource(R.string.statin_alert_statin_medicine))
+                        }
+                    },
+                    color = SimpleTheme.colors.material.error,
+                    style = SimpleTheme.typography.material.body2,
+                )
+            }
+        }
+    }
+}
+
+@Composable
+fun RiskProgressBar() {
+    val riskColors = listOf(
+        Color(0xFF00B849), // Very Low
+        Color(0xFFFFC800), // Low
+        SimpleTheme.colors.material.error, // Medium
+        Color(0xFFB81631), // High
+        Color(0xFF731814)  // Very High
+    )
+
+    Box(
+        modifier = Modifier
+            .fillMaxWidth()
+            .height(14.dp)
+            .drawWithContent {
+                drawContent()
+
+                val widthPerSegment = size.width / riskColors.size
+
+                drawLine(
+                    color = Color(0xFF2F363D),
+                    start = Offset(2 * widthPerSegment, 0f),
+                    end = Offset(2 * widthPerSegment, size.height),
+                    strokeWidth = 2.dp.toPx()
+                )
+                drawLine(
+                    color = Color(0xFF2F363D),
+                    start = Offset(size.width, 0f),
+                    end = Offset(size.width, size.height),
+                    strokeWidth = 2.dp.toPx()
+                )
+            }
+    ) {
+        Row(
+            modifier = Modifier
+                .fillMaxSize(),
+            verticalAlignment = Alignment.CenterVertically
+        ) {
+            riskColors.forEachIndexed { index, color ->
+                Box(
+                    modifier = Modifier
+                        .weight(1f)
+                        .height(4.dp)
+                        .background(
+                            color,
+                            shape = when (index) {
+                                0 -> RoundedCornerShape(topStartPercent = 50, bottomStartPercent = 50)
```

Comment:
I am just thinking, what if we just applied clip using clip modifier to the row irrespective of position? Since we will overlay the line indicators, the end should be gone. Wdyt?

Reply:
@msasikanth I tried doing that, but it clips the second line indicators too as the indicator is drawn inside the row. So found this way to implement the rounded corners.